boards/nordic/nrf5340dk/nrf5340dk_nrf5340_cpuapp_defconfig

```diff
@@ -18,3 +18,11 @@ CONFIG_SERIAL=y
 # enable console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# Enable RNG
+CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG=y
+CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG_ALLOW_NON_CSPRNG=y
+
+# Enable default settings
+CONFIG_NET_TCP_ISN_RFC6528=n
+CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_MBEDTLS_PSA=y
```

Comment:
this might throw a warning if Wi-Fi isn't enabled, and IMHO we don't need PSA for a shield, so, this has to be for nRF7002DK and not nRF5340.

Reply:
I have moved all the configuration to nrf7002dk as RNG source is required during PSA  init